### PR TITLE
Use default shape when car model cannot be loaded ref #10895

### PR DESCRIPTION
**Pull request by m-kro**
Since removal of the big bubbles in #10227 , there was no fallback solution when a given car model file could not be loaded. Only an error message was issued. Now a box shape is drawn for vehicles with failing car model file. This should help in cases like #10895. 

Signed-off-by: m-kro <m.barthauer@t-online.de>

### CODE REVIEW

**Comment by behrisch** [comment]
Could we have something which still allows to identify the orientation (similar to the default triangle in 2D)?

**Comment by m-kro** [comment]
Well the only predefined matching shape in this respect is a cone. I'll try it out.

File: src/osgview/GUIOSGBuilder.cpp [hunk 1]
```diff
@@ -379,7 +379,13 @@ GUIOSGBuilder::buildMovable(const MSVehicleType& type) {
     if (myCars.find(osgFile) == myCars.end()) {
         myCars[osgFile] = osgDB::readNodeFile(osgFile);
         if (myCars[osgFile] == 0) {
-            WRITE_ERROR("Could not load '" + osgFile + "'.");
+            WRITE_ERROR("Could not load '" + osgFile + "'. The model is replaced by a box shape.");
```
Comment (behrisch) [inline src/osgview/GUIOSGBuilder.cpp:382]: actually the error message is now wrong it is not a box anymore

Comment (m-kro) [reply src/osgview/GUIOSGBuilder.cpp:382]: Was corrected in commit https://github.com/eclipse/sumo/commit/1a8d867cbe00b3991ff7e213848a1e72958452a2

**Pull request merged**